site/docs/community.md

```diff
@@ -49,8 +49,9 @@ Issues are tracked in GitHub:
 
 ## Slack
 
-You can join the community via the [apache-iceberg](https://apache-iceberg.slack.com/) Slack workspace.
-If you encounter problems using it, please let us know by sending an email to <dev@iceberg.apache.org>.
+We use the [Apache Iceberg workspace](https://apache-iceberg.slack.com/) on Slack. To be invited, follow [this invite link](https://join.slack.com/t/apache-iceberg/shared_invite/zt-tlv0zjz6-jGJEkHfb1~heMCJA3Uycrg).
+
+Please note that this link may occasionally break when Slack does an upgrade. If you encounter problems using it, please let us know by sending an email to <dev@iceberg.apache.org>.
```

Comment:
These links work for 2k people and then expire. See https://slack.com/help/articles/201330256-Invite-new-members-to-your-workspace